### ISSUE TITLE
Icinga2 complains of syntax error in mail-host-notification.sh even though there isn't one

### ISSUE THREAD

**Issue by brando56894** [Icinga/icinga2#7607]
## Problem Description

Icinga2 complains of syntax error in mail-host-notification.sh even though it executes perfectly as my user, root, and the user (nagios) that Icinga is run as when executed manually.

```
root@pi:/etc/icinga2/scripts # icinga2 daemon -C
[2019-11-06 02:45:57 -0500] information/cli: Icinga application loader (version: r2.11.2-1)
[2019-11-06 02:45:57 -0500] information/cli: Loading configuration file(s).
[2019-11-06 02:45:57 -0500] critical/config: Error: syntax error, unexpected '[', expecting '('
Location: in /etc/icinga2/scripts/mail-host-notification.sh: 9:4-9:4
/etc/icinga2/scripts/mail-host-notification.sh(7): MAILBIN="mail"
/etc/icinga2/scripts/mail-host-notification.sh(8): 
/etc/icinga2/scripts/mail-host-notification.sh(9): if [ -z "`which $MAILBIN`" ] ; then
                                                      ^
/etc/icinga2/scripts/mail-host-notification.sh(10):   echo "$MAILBIN not found in \$PATH. Consider installing it."
/etc/icinga2/scripts/mail-host-notification.sh(11):   exit 1
```

## To Reproduce

1. Include the scripts directory by adding that path into *icinga.conf* (include "scripts/*.sh")
2. Execute *icinga2 daemon -C* or restart the Icinga2 daemon

## Expected behavior

The config check doesn't fail and/or the daemon restarts successfully

## Your Environment

* Version used (`icinga2 --version`): r2.11.2-1
* Operating System and version: Raspbian GNU/Linux v10 (Buster)
* Enabled features (`icinga2 feature list`): api checker command ido-mysql mainlog notification
* Icinga Web 2 version and modules (System - About): N/A
* Config validation (`icinga2 daemon -C`): See above under *Problem Description*

## More Info

This isn't specific to Raspbian because I just did the above on my server, which running an x86_64 version of Arch Linux and it give the same error.

**Comment by mcktr** [comment]
Hi,

this is a configuration issue, you don't need to include the script directory. You use the notification scripts with an `NotificationCommand`. In the definition of the `NotificationCommand` you also set the path to the script to execute. 

You can find an example inside the default confiuration in `conf.d/commands.conf` or in the documentation: https://icinga.com/docs/icinga2/latest/doc/09-object-types/#notificationcommand

For further help regarding to configuration please join our community plattform: https://community.icinga.com/

Best regards
Michael

**Comment by mcktr** [comment]
I just saw that you are coming from here: https://community.icinga.com/t/script-error-in-mail-host-notifications-sh/2669

I will join the conversation over there.

**Comment by brando56894** [comment]
Oh ok. I'll admit I find the documentation of Icinga to be very confusing and generally lacking for someone that is starting fresh, I'm attempting to learn this for my job, and we would like to use the Director but there is practically zero documentation on how to use it other than "here's how to set it up". I actually learned a lot more from random blogs than I did from reading the official documentation.
